### ISSUE TITLE
Issue with CODESTYLE.md#classes

### ISSUE THREAD

**Issue by animojis** [matter-lib/matter#3]
The **first** example for [classes](https://github.com/matter-lib/matter/blob/master/CODESTYLE.md#classes) is incorrect, and will not compile. Instead of

```cpp
class Person
{
public:
    int age;
}
```

It should be
```cpp
class Person
{
public:
    int age;
};
```

A minor fix, but this could confuse beginners to C++ and so it would be good advice to change the incorrect example to the fix i given.

**Comment by harrego** [comment]
Totally agree, will change soon in a new branch

**Comment by harrego** [comment]
Fixed in PR #4
